MAINTAINERS.md

```diff
@@ -168,6 +168,7 @@ Team which has full maintainer access to the Taxonomy repository
 - [`kelbrown20`](https://github.com/kelbrown20)
 - [`luke-inglis`](https://github.com/luke-inglis)
 - [`mingxzhao`](https://github.com/mingxzhao)
+- [`mrutkows`](https://github.com/mrutkows)
```

Comment:
@russellb Is this still temporary?

https://github.com/instructlab/community/pull/323#issuecomment-2177365699

Reply:
@vishnoianil If you remove this line and amend your commit, I can approve.

Reply:
Yep, waiting on @mrutkows to tell me when his demo is completed.

Reply:
@bjhargrave This was generated through the maintainers.py tool, so i didn't manually update it. Let me fix it and push the change.

Reply:
Fixed it.

Reply:
You removed the wrong line. Matt stays on line 36 but not 170.

Reply:
yup, realized it after the push :-) . fixed it.